src/C-interface/dense/bml_inverse_dense_typed.c

```diff
@@ -0,0 +1,63 @@
+#include "../blas.h"
+#include "../lapack.h"
+#include "../typed.h"
+#include "../bml_logger.h"
+#include "bml_inverse.h"
+#include "bml_allocate.h"
+#include "bml_parallel.h"
+#include "bml_inverse_dense.h"
+#include "bml_types.h"
+#include "bml_types_dense.h"
+
+#include <stdlib.h>
+#include <string.h>
+
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+
+/** Matrix inverse.
+ *
+ * \f$ B \leftarrow A^-1 f$
+ *
+ * \ingroup inverse_group
+ *
+ * \param A Matrix A
+ * \param B Inverse of Matrix A
+ */
+void TYPED_FUNC(
+    bml_inverse_dense) (
+    const bml_matrix_dense_t * A,
+    bml_matrix_dense_t * B)
+{
+    memcpy(B->matrix, A->matrix, A->N * A->N * sizeof(REAL_T));
```

Comment:
Without a check whether `B` has the right size (i.e. matches the size of `A`) this is not safe.